.github/workflows/unit_tests.yml

```diff
@@ -0,0 +1,45 @@
+name: Unit testing with Codecov coverage checking
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+      - main
+      - develop
+  pull_request:
+
+jobs:
+  job_unit_test:
+    name: Unit test
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/ledgerhq/ledger-app-builder/ledger-app-builder-lite:latest
```

Comment:
Does it work on lite? glibc is not needed?

Reply:
Apparently it works, I did not modify this CI content in any way..

Reply:
@lpascal-ledger do you know why?

Reply:
Why not? The test compilation is pretty simple and standard. Alpine has a `libc`, just not the `glibc`.
I'm guessing tools not working on alpine are using non-standard but common features of the `glibc` only.

Reply:
ok nice, thanks!